package.json

```diff
@@ -43,15 +43,19 @@
     "@buerli.io/react-cad": "0.9.0",
     "@react-three/drei": "9.77.1",
     "@react-three/fiber": "8.13.0",
-    "@react-three/postprocessing": "2.14.9",
+    "@react-three/postprocessing": "2.15.0",
     "antd": "^4.23.4",
     "arraybuffer-loader": "1.0.8",
     "react": "18.0.0",
     "react-dom": "18.0.0",
+    "react-is": "^16.8.0",
     "three": "0.153",
+    "three-stdlib": "2.23.5",
```

Comment:
Were these missing dependencies?

Reply:
@STINGNAILS  What is react-is used for?

Reply:
It is a peer dependency in styled-components. It isn't used anywhere directly in buerligons, but has to be specified for the corresponding warning to be gone. I've checked, it isn't listed in styled-components `>= 6.0 versions` dependencies, so I will update it tomorrow

Reply:
Ok, makes sense. But as we discussed in Slack. Because of node.js incompatibilities, please keep the styled-components version and add react-js to the dependencies as you already had it.